docker-compose.yml

```diff
@@ -55,29 +55,12 @@
 
 version: '3.5'
 
-volumes:
```

Comment:
Created a follow-up to remove the `env` boilerplates  from the github workflow files:https://issues.apache.org/jira/browse/ARROW-8713